internal/cmd/connect/command_custom_plugin.go

```diff
@@ -0,0 +1,44 @@
+package connect
+
+import (
+	v1 "github.com/confluentinc/cli/internal/pkg/config"
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+)
+
+type customPluginCommand struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type customPluginOut struct {
+	Id                        string   `human:"ID" serialized:"id"`
+	Name                      string   `human:"Name" serialized:"name"`
+	Description               string   `human:"Description" serialized:"description"`
+	ConnectorClass            string   `human:"Connector Class" serialized:"connector_class"`
+	ConnectorType             string   `human:"Connector Type" serialized:"connector_type"`
+	SensitiveConfigProperties []string `human:"Sensitive Config Properties" serialized:"sensitive_config_properties"`
```

Comment:
`confluent pipeline update` has a similar field called "Secret Names". Can we imitate that or come up with a shorter name?

Reply:
Since connect has been calling it sensitive properties for connectors for a long time it might be best to keep the name consistent because of user behavior. I have removed config from the name but if you still think the name is too big I can check in with our pm.